language-sql.md

```diff
@@ -3,11 +3,33 @@
 ### Minimum Requirements
 
 - MySQL keywords in uppercase
-- Column names in lowercase
+    E.g.: 
+    ```
+    UPDATE `consumer` SET `PASSWORD` = MD5(`PASSWORD`);
+    ```
+
+- Database table names must be plural. 
+    E.g.: 
+    `users`, `subscriptions`
+    
+    for Linking tables :
+    `users_subscriptions`
```

Comment:
I like the way @KixPanganiban explains it and it’s a valid point. With many people from different culture and various knowledge in English, going all singular might be less error-prone.

Usual spelling mistakes like “berrys”,  or “criterions” (if you have better examples, I’m OK to edit this comment :) ) would be avoided this way.

I like rules a lot, but simplicity is also a good thing, especially when naming things.